### PR TITLE
[MXNet][build] Relax pinned versions for requirements

### PR DESCRIPTION
**Pull request by waytrue17**
Relax pinned versions for MXNet DLC requirements

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

## Label Checklist
- [x] I have added the project label for this PR (*<project_name>* or "Improvement")

## PR Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] If the PR changes affects SM test, I've modified dlc_developer_config.toml in my PR branch by setting sagemaker_tests = true and efa_tests = true
- [ ] If this PR changes existing code, the change fully backward compatible with pre-existing code. (Non backward-compatible changes need special approval.)
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `ei_mode = true` or `neuron_mode = true`

#### Benchmark Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `benchmark_mode = true`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: mxnet/training/docker/1.6.0/py2/Dockerfile.cpu [hunk 1]
```diff
@@ -98,17 +98,17 @@ WORKDIR /
 # python-dateutil==2.8.0 to satisfy botocore
```
Comment (arjkesh) [inline mxnet/training/docker/1.6.0/py2/Dockerfile.cpu:98]: Note: buildspec is currently set to MX1.8 only, so this PR will only build/test MX 1.8 images

Comment (waytrue17) [reply mxnet/training/docker/1.6.0/py2/Dockerfile.cpu:98]: Are you suggesting that I can only update MX1.8 in this PR?

Comment (arjkesh) [reply mxnet/training/docker/1.6.0/py2/Dockerfile.cpu:98]: Hey all, you can add diff versions to buildspec, but if planning on making this change for several version, I'd suggest openings separate PRs for ease of iteration and in order to debug issues separately

**Comment by waytrue17** [comment]
@arjkesh @saimidu Could you please review?

**Review by saimidu — APPROVED** [review]

**Review by josephevans — APPROVED** [review]
Thanks Wei. Looks good.

**Comment by saimidu** [comment]
Thanks for the review @josephevans . Merging this PR.

**Pull request merged**